app/AppComponents.scala

```diff
@@ -43,8 +43,7 @@ class AppComponents(context: Context, identity: AppIdentity, creds: AWSCredentia
 
   val credentials = configuration.get[String]("typerighter.google.credentials")
   val spreadsheetId = configuration.get[String]("typerighter.sheetId")
-  val range = configuration.get[String]("typerighter.sheetRange")
-  val ruleResource = new SheetsRuleResource(credentials, spreadsheetId, range)
+  val ruleResource = new SheetsRuleResource(credentials, spreadsheetId)
```

Comment:
Why is the range no longer coming for the configuration?

Reply:
Ah, yes, should have flagged – because it's tied to the ranges we expect from our columns, which are determined in code. In retrospect, I don't think this should have been parameterised to begin with.